etebase_server/settings.py

```diff
@@ -167,8 +169,20 @@
 
     if "database-options" in config:
         DATABASES["default"]["OPTIONS"] = config["database-options"]
-
-ETEBASE_CREATE_USER_FUNC = "etebase_server.django.utils.create_user_blocked"
+        
+    if "ldap" in config:
+        ldap = config["ldap"]
+        LDAP_SERVER = ldap.get("server", "")
+        LDAP_SEARCH_BASE = ldap.get("search_base", "")
+        LDAP_FILTER = ldap.get("filter", "")
+        LDAP_BIND_DN = ldap.get("bind_dn", "")
+        LDAP_BIND_PW = ldap.get("bind_pw", "")
+        LDAP_BIND_PW_FILE = ldap.get("bind_pw_file", "")
+        LDAP_CACHE_TTL = ldap.get("cache_ttl", "")
```

Comment:
Should these really have empty strings as defaults? Wouldn't it make more sense to error if they aren't specified (or for `bind_pw`/`bind_pw_file`, if neither of them is specified)?